database.aml

```diff
@@ -227,3 +330,7 @@ logs | Represents a operation log
   type varchar(255) | Type of the log (sale, transaction, product movement, etc.) [hardcoded enum]
   state varchar(255) | State of the log (created, processed, etc.) [hardcoded enum]
   message text nullable | Message of the log if needed (error message for example)
+  organization_id int | ID of the organization who own the log
```

Comment:
The organization can be null, that would allow the platform to perform and store global logs